account_invoice_ubl_email_attachment/tests/test_ubl_email_attachment.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2019 Onestein (<https://www.onestein.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import HttpCase
+from odoo.tools import mute_logger
+
+
+class TestUblInvoiceEmailAttachment(HttpCase):
+    def setUp(self):
```

Comment:
can you please use `HttpSavepointCase` and move setup code to `setUpClass`? The code here runs for _every_ test and I don't think is required.

Reply:
@simahawk Done.
Apparantly GitHub's fetch upstream added a merge-commit. Do I need to squash?

Reply:
you should rebase instead of merging. Just run `git pull --rebase OCA 14.0` when you have to sync from main branch.

Also, now this seems to be a migration: the history is missing. Please follow these instructions https://github.com/OCA/web/issues/1695

Reply:
Sorry for that. Any suggestion how to solve this `git` line, PR ?

Reply:
I was too quick with the GitHub fetching action. 
I assumed (not good) GitHub would rebase my fork onto OCA's branch. Sorry for not grasping this and the migration procedure.
Let me know if I need to fix the history and start a new PR with correct baseline/history.

Reply:
I would start from scratch. Just reset hard on 14.0, follow the instruction on https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0 and cherry-pick your good commits (or redo migration changes).